themes/fino.zsh-theme

```diff
@@ -11,26 +11,35 @@
 #
 # Also borrowing from http://stevelosh.com/blog/2010/02/my-extravagant-zsh-prompt/
 
+function virtualenv_prompt_info(){
+  [[ -n ${VIRTUAL_ENV} ]] || return
+  echo "${ZSH_THEME_VIRTUALENV_PREFIX:=[}${VIRTUAL_ENV:t}${ZSH_THEME_VIRTUALENV_SUFFIX:=]}"
+}
+
 function prompt_char {
   git branch >/dev/null 2>/dev/null && echo "±" && return
   echo '○'
 }
 
 function box_name {
-    [ -f ~/.box-name ] && cat ~/.box-name || echo ${SHORT_HOST:-$HOST}
+  [ -f ~/.box-name ] && cat ~/.box-name || echo ${SHORT_HOST:-$HOST}
 }
 
-local ruby_env='using%{$FG[243]%} $(ruby_prompt_info)'
+local ruby_env='$(ruby_prompt_info)'
 local git_info='$(git_prompt_info)'
 local prompt_char='$(prompt_char)'
+local virtualenv_info='$(virtualenv_prompt_info)'
+
 
 
-PROMPT="╭─%{$FG[040]%}%n%{$reset_color%} %{$FG[239]%}at%{$reset_color%} %{$FG[033]%}$(box_name)%{$reset_color%} %{$FG[239]%}in%{$reset_color%} %{$terminfo[bold]$FG[226]%}%~%{$reset_color%}${git_info} %{$FG[239]%}${ruby_env}
+PROMPT="╭─%{$FG[040]%}%n%{$reset_color%} %{$FG[239]%}at%{$reset_color%} %{$FG[033]%}$(box_name)%{$reset_color%} %{$FG[239]%}in%{$reset_color%} %{$terminfo[bold]$FG[226]%}%~%{$reset_color%}${git_info} ${ruby_env}${virtualenv_info}
 ╰─${prompt_char}%{$reset_color%} "
 
 ZSH_THEME_GIT_PROMPT_PREFIX=" %{$FG[239]%}on%{$reset_color%} %{$fg[255]%}"
 ZSH_THEME_GIT_PROMPT_SUFFIX="%{$reset_color%}"
 ZSH_THEME_GIT_PROMPT_DIRTY="%{$FG[202]%}✘✘✘"
 ZSH_THEME_GIT_PROMPT_CLEAN="%{$FG[040]%}✔"
-ZSH_THEME_RUBY_PROMPT_PREFIX="‹"
+ZSH_THEME_RUBY_PROMPT_PREFIX="%{$FG[239]%}using%{$FG[243]%} ‹"
 ZSH_THEME_RUBY_PROMPT_SUFFIX="›%{$reset_color%}"
+ZSH_THEME_VIRTUALENV_PREFIX="%{$FG[239]%}using%{$FG[243]%} ‹"
+ZSH_THEME_VIRTUALENV_SUFFIX="›%{$reset_color%}"
```

Comment:
Maybe use a different formatting for virtualenv to distinguish it from the ruby prompt?

Reply:
Also, here yes should export `VIRTUAL_ENV_DISABLE_PROMPT=1`, since it would be displayed by the theme after these changes are merged.

Reply:
> Maybe use a different formatting for virtualenv to distinguish it from the ruby prompt?

Maybe use a double guillemet?
![image](https://user-images.githubusercontent.com/26824900/95000360-c6182000-05c8-11eb-9c30-3b8731165e8c.png)

Reply:
> Also, here yes should export VIRTUAL_ENV_DISABLE_PROMPT=1, since it would be displayed by the theme after these changes are merged.

Done